index.js

```diff
@@ -14,37 +14,31 @@
 
 const core = require('@actions/core');
 const exec = require('@actions/exec');
+const shellParse = require('shell-quote/parse')
 
 /**
  * Execute pylint and report issues, if any found.
  */
 function run() {
-    const pylintDisable = core.getInput('pylint-disable') || "";
```

Comment:
[fix] This was used further down, see [non-failing failure here](https://github.com/foundrytom/OpenAssetIO/actions/runs/6381077087/job/17316792095?pr=44#step:6:42).

Reply:
:open_mouth: how did I miss this! Good spot.

Took the opportunity to force push with a tweaked commit message (got rid of the `[CI]` tag).

Showing it working in https://github.com/OpenAssetIO/OpenAssetIO/actions/runs/6381982980/job/17319617865?pr=1133